protocol-designer/src/pages/CreateNewProtocolWizard/SelectPipettes.tsx

```diff
@@ -127,9 +113,9 @@ export function SelectPipettes(props: WizardTileProps): JSX.Element | null {
       pipettesByMount.right.tiprackDefURI == null)
 
   const isDisabled =
-    (page === 'add' && pipettesByMount[defaultMount].tiprackDefURI == null) ||
-    noPipette ||
-    selectedValues.length === 0
+    page === 'add' &&
+    pipettesByMount[defaultMount].tiprackDefURI == null &&
+    noPipette
```

Comment:
for the attached screenshot case

```suggestion
 const isDisabled =
    (page === 'add' &&
      pipettesByMount[defaultMount].tiprackDefURI == null &&
      noPipette) ||
    selectedValues.length === 0
```

Reply:
this results in requiring to add a 2nd pipette when you go back to add a 2nd pipette. let me look into another option

Reply:
ok i think i found a way that works!